plugin.rb

```diff
@@ -11,6 +11,28 @@
 DiscoursePluginRegistry.serialized_current_user_fields << "nationalflag_iso"
 
 after_initialize do
+
+  module ::DiscourseNationalFlags
+    class Engine < ::Rails::Engine
+      engine_name "discourse_national_flags"
+      isolate_namespace DiscourseNationalFlags
+    end
+  end
+
+  load File.expand_path('../lib/flags.rb', __FILE__)
+
+  require_dependency 'about_controller'

```

Comment:
@merefield require_dependency is no longer needed after the zeitwork upgrade in core Discourse.

Reply:
Removed, thanks